tracer/build/_build/Build.Gitlab.cs

```diff
@@ -17,173 +19,69 @@
-        const string timestampServer = "http://timestamp.digicert.com/";
-
-        try
+        var jarSignJar = Environment.GetEnvironmentVariable("JARSIGN_JAR");
+        if (string.IsNullOrEmpty(jarSignJar))
         {
-            var (certPath, certPassword) = UseTestPfxCertificate
-                                               ? (@"test_cert.pfx", "Passw0rd")
-                                               : await GetSigningMaterial(tempFileName);
-
-            Logger.Information("Signing material retrieved");
-
-            var binaries = filesToSign
-               .Where(x => !x.ToString().EndsWith(".nupkg"))
-                .ToList();
-
-            if (binaries.Any())
-            {
-                Logger.Information("Signing binaries...");
-                binaries.ForEach(file => SignBinary(certPath, certPassword, file));
-                Logger.Information("Binary signing complete");
-            }
-
-            var nupkgs = filesToSign
-                        .Where(x => x.ToString().EndsWith(".nupkg"))
-                        .ToList();
-
-            if (nupkgs.Any())
-            {
-                Logger.Information("Signing NuGet packages...");
-                nupkgs.ForEach(file => SignNuGet(certPath, certPassword, file));
-                Logger.Information("NuGet signing complete");
-            }
+            throw new Exception("Environment variable 'JARSIGN_JAR' missing");
         }
-        finally
-        {
-            File.Delete(tempFileName);
-        }
-
-        return;
 
-        void SignBinary(string certPath, string certPassword, AbsolutePath binaryPath)
-        {
-            Logger.Information($"Signing {binaryPath}");
-
-            SignToolTasks.SignTool(
-                x => x
-                    .SetFiles(binaryPath)
-                    .SetFile(certPath)
-                    .SetPassword(certPassword)
-                    .SetTimestampServerUrl(timestampServer)
-            );
-        }
+        Logger.Information("Signing {Count} binaries...", filesToSign.Count);
+        filesToSign.ForEach(file => SignBinary(file, jarSignJar));
+        Logger.Information("Binary signing complete");
 
-        void SignNuGet(string certPath, string certPassword, AbsolutePath binaryPath)
+        void SignBinary(AbsolutePath binaryPath, string jarSignJar)
         {
-            Logger.Information($"Signing {binaryPath}");
+            Logger.Information("Signing {BinaryPath}", binaryPath);
 
-            // nuke doesn't expose the sign tool
-            try
-            {
-                NuGetTasks.NuGet(
-                    $"sign \"{binaryPath}\"" +
-                    $" -CertificatePath {certPath}" +
-                    $" -CertificatePassword {certPassword}" +
-                    $" -Timestamper {timestampServer} -NonInteractive",
+            var signProcess = ProcessTasks.StartProcess(
+                    "dd-wcs",
+                    $"sign --jsign-jar {jarSignJar} {binaryPath}",
```

Comment:
the `--jsign-jar {jarSignJar}` shouldn't be required.  The idea is that it's all wrapped up in the `dd-wcs` wrapper

Reply:
Ah ok, [the linked PR](https://github.com/DataDog/synthetics-worker/pull/4063) provides these arguments - should that be updated too? I'll update and test it

Reply:
Cool, that worked, done in https://github.com/DataDog/dd-trace-dotnet/pull/4374/commits/ccde701d51f6eedf942c426a6053e02c34d38d4a